source/includes/gcp/_vpn_tunnels.md

```diff
@@ -185,9 +185,9 @@ Optional | &nbsp;
 `description`<br/>*string* | An optional description.
 `localTrafficSelector`<br/>*string* | Local traffic selector to use when establishing the VPN tunnel with the peer VPN gateway. The value should be a CIDR formatted string.
```

Comment:
this line still says gateway ;)

Reply:
If I am not mistaken the use of vpnGateway is correct here. The tunnel is connecting to the peer-gateway.

Reply:
okie dokie ! :D you know more than me!